pkg/internalapis/cnsvolumeinfo/v1alpha1/cnsvolumeinfo_types.go

```diff
@@ -28,6 +28,12 @@ type CNSVolumeInfoSpec struct {
 
 	// vCenterServer is the IP/FQDN of the vCenter host on which the CNS volume is accessible.
 	VCenterServer string `json:"vCenterServer"`
+
+	// ID of the storage policy
+	StoragePolicyId string `json:"storagePolicyId,omitempty"`
```

Comment:
do we need to regenerate CRD at pkg/internalapis/cnsvolumeinfo/config/cns.vmware.com_cnsvolumeinfoes.yaml with this field change in CnsVolumeInfo API

Reply:
Yes, I have pushed the CRD updates. PTAL.